File: __tests__/__fixtures__/output/python/python3/text-plain.py
import http.client

conn = http.client.HTTPSConnection("httpbin.org")

payload = "Hello World"

headers = { 'content-type': "text/plain" }

conn.request("POST", "/anything", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: src/targets/node/axios/fixtures/text-plain.js
import axios from 'axios';

const options = {
  method: 'POST',
  url: 'https://httpbin.org/anything',
  headers: {'content-type': 'text/plain'},
  data: 'Hello World'
};

axios
  .request(options)
  .then(res => console.log(res.data))
  .catch(err => console.error(err));